pkg/ccl/backupccl/show.go

```diff
@@ -486,6 +486,9 @@ func checkBackupFiles(
 	storeFactory cloud.ExternalStorageFromURIFactory,
 	user username.SQLUsername,
 ) ([][]int64, error) {
+	maxMissingFiles := 10
+	missingFiles := make(map[string]bool, maxMissingFiles)
+	numMissingFiles := 0
```

Comment:
I think you can just use len() on the map?

Reply:
Done.